ext/mbstring/tests/mb_str_split.phpt

```diff
@@ -0,0 +1,34 @@
+--TEST--
+mb_str_split()
+--SKIPIF--
+<?php extension_loaded('mbstring') or die('skip mbstring not available'); ?>
+--INI--
+output_handler=
+mbstring.func_overload=0
+--FILE--
+<?php
+ini_set('include_path','.');
+include_once('common.inc');
+
+// EUC-JP
+$utf8_jp = "どうも";
+$euc_jp = mb_convert_encoding($utf8_jp, "EUC-JP", "UTF-8");
+
+// UTF-8 russian
+$utf8_ru_1 = "Путин";
+$utf8_ru_2 = "вор";
```

Comment:
@legale let's leave politics aside.

Reply:
sorry, could not resist.

Reply:
@legale Please, if you could use other Russian words, rather than these two.

Reply:
ok, no problem.